### PR TITLE
Feature/focuspoint include

### PR DESCRIPTION
**Pull request by HarwinBorger**
Contains also feature/jquery focuspoint

### CODE REVIEW

**Comment by DannyvanHolten** [comment]
Address the codeclimate issue please :)

P.S: This might be autofixed by using SASS.

**Comment by DannyvanHolten** [comment]
Let's keep the renaming for one branch only. For now keep it to image-focus to keep our history clean.

P.S: I took the liberty to change this myself for now.

**Comment by DannyvanHolten** [comment]
Please use SASS for styling. Grunt will minify all files necessary. The scss file will be include in the github repo and will be excluded of the zip package by .gitattributes. But as we need it for development we need to have that SCSS in the repo.

**Comment by DannyvanHolten** [comment]
Use spaces instead of tabs in PHP please :)

Edit: Fixed this in your branch

**Comment by DannyvanHolten** [comment]
Please use: add_action('admin_enqueue_scripts', [$this, 'loadScripts']); for loading the Scripts.

Edit: Fixed this in your branch

**Comment by DannyvanHolten** [comment]
Please create a seperate Class for the FocusPoint functionality

**Comment by DannyvanHolten** [comment]
Please only load the script on the screens where they are needed.

**Comment by DannyvanHolten** [comment]
Please use IMAGEFOCUS_ASSETS instead of basename(__FILE__) when enqeueing the scripts.

File: js/image-focus.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+			var focusX = (offsetX / imageW - .5) * 2;
+			var focusY = (offsetY / imageH - .5) * -2;
+
+			//Calculate CSS Percentages
+			var percentageX = (offsetX / imageW) * 100;
+			var percentageY = (offsetY / imageH) * 100;
+
+			console.log('FocusX:' + focusX.toFixed(2) + ', FocusY:' + focusY.toFixed(2));
+
+			$('.' + css.wpimageFocus.point).css({
```
Comment (HarwinBorger) [inline js/image-focus.js]: replacement of wpimageFocus to imageFocus forgotten

**Pull request merged**